create-auth/index.js

```diff
@@ -0,0 +1,45 @@
+import { createStore } from '@logux/state'
+
+export function createAuth(client) {
+  let auth = createStore(() => {
+    auth.set({
+      userId: client.options.userId,
+      isAuthenticated: client.node.state === 'synchronized'
+    })
+
+    let stateBinded = false
+    let unbindState
+
+    let bindState = () => {
+      stateBinded = true
+      unbindState = client.node.on('state', () => {
+        if (client.node.state === 'synchronized') {
+          auth.set({ ...auth.value, isAuthenticated: true })
+          unbindState()
+          stateBinded = false
+        }
+      })
+    }
+
+    bindState()
+
+    let unbinds = [
+      client.node.catch(error => {
```

Comment:
Let’s use normal variables like `unbindError` since we have only 2 event listeners (and static list)